src/Symfony/Component/Serializer/Tests/SerializerTest.php

```diff
@@ -893,7 +893,9 @@ public function testCollectDenormalizationErrors(?ClassMetadataFactory $classMet
 
             $this->fail();
         } catch (\Throwable $th) {
+            /** @var PartialDenormalizationException $th */
```

Comment:
Before the assert, we aren't sure that `$th` is a `PartialDenormalizationException` (same for the next occurrence)